src/devServer.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import express from 'express';
+import { renderToString } from 'react-dom/server';
+
+const resolveRootComponent = () => require('app/components/Root').default;
```

Comment:
Obviously we enforce strict directory structure. We should as well check, whether the directory exists.